app/styles/component_tree.scss

```diff
@@ -16,50 +15,58 @@
   &:hover {
     background-color: var(--spec06);
 
-    .component-tree-item__view-element { opacity: 1; }
+    .component-tree-item__view-element {
+      opacity: 1;
+    }
   }
 }
 
 .component-tree-item__view-element {
-  display: flex;
   align-items: center;
+  background: transparent;
+  border: 0;
+  display: flex;
   height: 100%;
   padding: 0;
-  border: 0;
-  background: transparent;
 
-  &:focus { outline: none; }
+  &:focus {
+    outline: none;
+  }
 }
 
 .component-tree-item__tag {
-  display: flex;
   cursor: default;
+  display: flex;
 }
-  .component-tree-item--component .component-tree-item__tag {
-    cursor: pointer;
-  }
 
-.component-tree-item__bracket::before {
-  content: '{{';
+.component-tree-item--component .component-tree-item__tag {
+  cursor: pointer;
+}
+
+.component-tree-item__bracket:before {
   color: var(--base07);
+  content: '{{';
 }
-.component-tree-item__bracket::after {
-  content: '}}';
+
+.component-tree-item__bracket:after {
```

Comment:
Minor: psuedo elements in CSS should use `::`. It's in more places, used for `before` and `after`.

Reply:
The stylelint rules have a rule to use a single `:`. I prefer the single, but not a big deal either way to me.